### PR TITLE
`appendControl` adds controls with `alwaysOpen` enabled. Closes #37

### LINKED ISSUE #37: Issue with `alwaysOpen` being limiting.
## New issue checklist

- [x] I have read all of the [`README`](../README.md) 
- [X] I have searched [existing issues](https://github.com/opentok/accelerator-textchat-js/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

### General information

- Library version(s): 1.0.30
- iOS/Android/Browser version(s): Chrome 80.0.3987.163
- Devices/Simulators/Machine affected: N/A
- Reproducible in the demo project? (Yes/No): N/A
- Related issues: N/A

#### Expected behavior

> N/A

#### Actual behavior

> N/A

#### Steps to reproduce

> N/A

#### Crash log? Screenshots? Videos? Sample project?

>N/A

## Question or Feature Request

In the accelerator pack there are currently two options:

1. `alwaysOpen: true` where the chat is automatically started and displayed, but the button for toggling the chat is hidden. 
2. `alwaysOpen: false` where the chat is NOT started and displayed until the chat button is clicked. The display can be toggled with the chat button

Is there a way to make it so that when the application begins, the chat is automatically started and the display is able to be toggled on and off?

My exact use case is that if two users are in the video call, I want them to be able to send and receive chats from the beginning of the call without needing either of them to click the button. However, I don't want the chat to show all of the time. I was going to show a notification ping when a chat is received. However, I cannot do that until the chat is manually turned on by the user.

### PR DESCRIPTION
**Pull request by MichaelJolley**
## Pull request checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I have resolved any merge conflicts. Confirmation: ____

## What's in this pull request?

Currently, when providing `alwaysOpen: true` the controls are always hidden. The `appendControl` option already exists and is used when `alwaysOpen: false`. This PR changes that behavior to allow `appendControl` to force adding the controls even when `alwaysOpen: true`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 
 --------------------------------------
+#### [1.0.31]
+
+[UPDATED] `appendControl` now allows controls to display when `alwaysOpen` is true
```
Comment (marinaserranomontes) [inline CHANGELOG.md:8]: Should we indicate the deps update as well?

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,11 @@
 language: node_js
 node_js:
-- '4.4'
+- 12
```
Comment (marinaserranomontes) [inline .travis.yml:3]: what do you think if we lock the node version with .nvmrc?

File: debug.log [hunk 3]
```diff
@@ -0,0 +1 @@
+[0811/195304.174:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (ggoldens) [inline debug.log:1]: Probably we want to remove this log from the repo and add this file to .gitignore

File: package.json [hunk 4]
```diff
@@ -26,34 +26,41 @@
...
+    "merge-stream": "^2.0.0",
+    "mocha": "^8.1.1",
     "npm-css": "^0.2.3",
-    "opentok-one-to-one-communication": "^1.0.6",
-    "replace": "^0.3.0",
-    "rimraf": "^2.4.3",
-    "sinon": "^1.17.0",
-    "uglify-js": "^2.8.15",
-    "yuidocjs": "^0.9.0"
+    "opentok-one-to-one-communication": "^1.0.27",
```
[the remark this answers is not in the document]
Comment (ggoldens) [reply package.json:53]: I think we can remove this dep.

File: CHANGELOG.md (hunk 1 above)
Comment (MichaelJolley) [reply CHANGELOG.md:8]: Added

File: debug.log (hunk 3 above)
Comment (MichaelJolley) [reply debug.log]: Removed and updated .gitignore to prevent it from being inadvertently re-added in the future.

**Comment by MichaelJolley** [comment]
Got some bad stuff in here so starting a new clean PR

**Pull request closed without merging**